hack/debug.sh

```diff
@@ -61,6 +68,15 @@ log_system () {
 }
 
 do_master () {
+    services=($aos_master_service $aos_master_controllers_service $aos_master_api_service)
+    start_args=("master" "master controllers" "master api")
+    for i in "${!services[@]}"; do
+        config=$(get_config_path_from_service "${start_args[$i]}" "${services[$i]}")
+        if [ -n "$config" ]; then
+            break
```

Comment:
What if 'master controller' and 'master api' services are running on the same host? This will only pick one config.